.github/workflows/services-all-features.yml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
 
 env:
-  RUSTFLAGS: --deny warnings --allow unused_attributes --allow unreachable-code --allow unused-assignments
```

Comment:
This workflow [fails](https://github.com/ctaggart/azure-sdk-for-rust/runs/4676293295?check_suite_focus=true) with this change. I'm going to revert it in my upcoming PR. I created #553 for removing the warnings from generated services.